aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/OnboardApplicationDeepAnalysisBuilder.java

```diff
@@ -142,7 +101,7 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath filePath, @Nonnull
             String caipVersion = app.getCaipVersion();
             String targetNode = app.getTargetNode();
             boolean verbose = getDescriptor().configuration.isVerbose();
-            OnboardApplicationDeepAnalysisBuilder.JnksLogPollingProviderImpl jnksLogPollingProvider = new JnksLogPollingProviderImpl(run, listener, verbose);
+            JnksLogPollingProviderServiceImpl jnksLogPollingProvider = new JnksLogPollingProviderServiceImpl(jobsService, run, listener, verbose, getSleepDuration());
```

Comment:
If "Jnks" stands for "Jenkins", I would write it "Jenkins" for class & member names